resources/waterheater.rb

```diff
@@ -575,15 +575,42 @@ def self.apply_heatpump(model, runner, space, weather, t_set, vol, ef,
   end
 
   def self.apply_indirect(model, runner, space, cap, vol, t_set, oncycle_p, offcycle_p, ec_adj,
-                          nbeds, boiler, boiler_plant_loop, boiler_fuel_type, dhw_map, sys_id, wh_type, jacket_r)
+                          nbeds, boiler, boiler_plant_loop, boiler_fuel_type, dhw_map, sys_id, wh_type, jacket_r, standby_loss)
     obj_name_indirect = Constants.ObjectNameWaterHeater
     convlim = model.getConvergenceLimits
     convlim.setMinimumPlantIterations(3) # add one more minimum plant iteration to achieve better energy balance across plant loops.
 
     if wh_type == "space-heating boiler with storage tank"
       tank_type = Constants.WaterHeaterTypeTank
+      # Actual tank volume = 95% nominal tank volume
+      act_vol = 0.95 * vol
```

Comment:
Please reuse the existing `calc_actual_tankvol` method here so we have consistency.